requirements.txt

```diff
@@ -1,2 +1,4 @@
 click
 requests
+scrapy
+six
```

Comment:
:+1: that's how I see it, it represents the things we use in development and run our tests against.

and in setup.py we leave the dependencies as loosely defined as possible.